src/components/Tests/TestTicket/index.tsx

```diff
@@ -64,7 +64,7 @@ export const TestTicket = (props: TestTicketProps) => {
       description={{
         content: renderDescription()
       }}
-      attachment={attachment}
+      attachments={[attachment]?.filter(Boolean).map((a) => a!)}
```

Comment:
u can extract type for attachment and use this construction 
`fiter((item): item is Attachment => !!item)`